Installation instructions fail on impish

The result of following the installation instructions on impish:
```
any_proxy.go:47:5: no required module provides package github.com/namsral/flag: go.mod file not found in current directory or any parent directory; see 'go help modules'
any_proxy.go:50:5: no required module provides package github.com/zdannar/flogger: go.mod file not found in current directory or any parent directory; see 'go help modules'

```
The new LTS on this code base is just around the corner

--- a/.gitignore
+++ b/.gitignore
@@ -2,6 +2,8 @@
 !.gitignore
 !*_test.go
 !any_proxy.go
+!go.mod
+!go.sum
 !make.bash
 !stats.go
 !sni.go

--- a/README.md
+++ b/README.md
@@ -16,7 +16,7 @@ For more info, see http://blog.rchapman.org/post/47406142744/transparently-proxy
 
 ## Maintenance
 
-This project is actively maintained.  As of this writing (March 2019), I haven't had any bugs submitted in a few
+This project is actively maintained.  As of this writing (November 2021), I haven't had many bugs submitted in a few
 years, which is why you don't see much for code changing.  But be assured that I am watching the project and will
 address any bugs that come in.
 

--- a/go.mod
+++ b/go.mod
@@ -1,0 +1,8 @@
+module github.com/ryanchapman/go-any-proxy
+
+go 1.17
+
+require (
+	github.com/namsral/flag v1.7.4-pre
+	github.com/zdannar/flogger v0.0.0-20160527010203-f7a4368a7b39
+)

